src/components/LandingPage.tsx

```diff
@@ -139,73 +139,73 @@ export function LandingPage() {
-              focus on building tools that deliver value.
+              Arcade solves the fundamental challenges that have limited AI
+              agents, empowering them to take meaningful actions in real-world
+              applications.
             </p>
           </div>
           <div className="mt-16 grid gap-8 md:grid-cols-2">
             <ChallengeSolution
               challenge={{
                 icon: Shield,
-                title: "Auth Management",
+                title: "Agents Can't Act on Users' Behalf",
                 description:
-                  "Managing authentication across services is complex, from OAuth flows and API keys to storing credentials, refreshing tokens, and keeping your AI agent connected to services.",
+                  "Today's agents can't securely access user data or perform actions as the user, limiting them to generic tools like search engines and calculators.",
               }}
               solution={{
                 icon: Key,
-                title: "Built-in Auth Management",
+                title: "Secure User Impersonation",
                 description:
-                  "Securely manages OAuth flows, API keys, and user tokens with automatic refresh and scoped access, allowing agents to retrieve and act on user-specific data seamlessly and securely.",
+                  "Arcade lets agents securely access user-specific services like email, calendars, and documents through built-in OAuth and token management.",
                 href: "/home/auth/how-arcade-helps",
               }}
             />
             <ChallengeSolution
               challenge={{
-                icon: Bot,
-                title: "AI Model Compatibility",
+                icon: RefreshCcw,
+                title: "Tools Can't Scale Independently",
                 description:
-                  "Each AI provider (OpenAI, Anthropic, etc.) has its own approach to function calling. You need a unified interface that works consistently across providers while maintaining flexibility to swap models.",
+                  "Tools run on the same resources as your application, creating performance bottlenecks and limiting what agents can do as demand grows.",
               }}
               solution={{
-                icon: Code,
-                title: "Flexible SDK",
+                icon: Zap,
+                title: "Tools That Scale Independently",
                 description:
-                  "Provides a unified SDK to create and execute tools seamlessly across LLM providers like OpenAI and Anthropic. Build once, integrate everywhere, with powerful evaluation and testing tools.",
-                href: "/home/supported-models",
+                  "Arcade separates tool execution from your application, allowing each tool to run on its optimal infrastructure and scale based on its specific needs.",
+                href: "/home/evaluate-tools/why-evaluate-tools",
```

Comment:
I think this page makes more sense based on the context: 
```suggestion
                href: "/home/use-tools/tools-overview",
```